test/e2e/containers/fake_datadog/app/api.py

```diff
@@ -20,7 +20,7 @@
 
 
 def get_collection(name: str):
-    c = pymongo.MongoClient("127.0.0.1", 27017, connectTimeoutMS=5000)
+    c = pymongo.MongoClient("192.168.254.241", 27017, connectTimeoutMS=5000)
```

Comment:
Do you still need this change since this https://github.com/DataDog/datadog-agent/pull/1902 was merged?

Reply:
it seems like the host is set to 0.0.0.0, I should be able to move back to 127.0.0.1"